GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -512,7 +513,12 @@ private void ResetSelectedItemsTo(bool resetToParent, bool resetAndDelete)
 
                     IReadOnlyList<GitItemStatus> resetItems = (resetToParent ? selectedItems.Items()
                         : selectedItems.Items().Select(item => item.InvertStatus())).ToList();
-                    Module.ResetChanges(id, resetItems, resetAndDelete: resetAndDelete, _fullPathResolver, filesInUse: out _, output: out _);
+                    Module.ResetChanges(id, resetItems, resetAndDelete: resetAndDelete, _fullPathResolver, out StringBuilder output);
+
+                    if (!string.IsNullOrEmpty(output.ToString()))
```

Comment:
`output.ToString()` is called multiple times. Please use https://learn.microsoft.com/dotnet/api/system.text.stringbuilder.length instead.

Reply:
ok